### PR TITLE
bpo-39654: pyclbr: remove old references to class browser & add expla…

### PR DESCRIPTION
**Pull request by hakancelikdev**
https://bugs.python.org/issue39654

### CODE REVIEW

File: Doc/library/pyclbr.rst [hunk 1]
```diff
@@ -29,6 +29,9 @@ modules.
    *path* is a sequence of directory paths prepended to ``sys.path``,
    which is used to locate the module source code.
 
+  This function is a filtered version kept for back compatibility which
```
Comment (isidentical) [inline Doc/library/pyclbr.rst]:
```suggestion
  This function is a filtered version kept for the backwards compatibility which
```

File: Doc/library/pyclbr.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
-:mod:`pyclbr` --- Python class browser support
+:mod:`pyclbr` --- Python module browser support
 ==============================================
```
Comment (isidentical) [inline Doc/library/pyclbr.rst]:
```suggestion
===============================================
```

File: Doc/library/pyclbr.rst (hunk 1 above)
Comment (terryjreedy) [reply Doc/library/pyclbr.rst]: 'the' is wrong.  'back' is as good as 'backwards'.  I rewrote these two lines.

**Review by terryjreedy — APPROVED** [review]
I will merge as amended when CI passes.

**Pull request merged**